### PR TITLE
lab 6 notes

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: Styles/main.css [hunk 1]
```diff
@@ -47,9 +47,10 @@ body {
 }
 
 h1 {
-    color: rgb(255, 0, 0);
+    color: rgb(255, 0, 0, 0);
     background: rgb(0, 9, 255);
     background: radial-gradient(circle, rgba(0, 9, 255, 0.5) 0%, rgba(255, 0, 0, 0.2) 100%);
+    background-clip: text;
```
Comment (rmartin-sc) [inline Styles/main.css:53]: Make text transparent and set background-clip to text for the full effect

File: Styles/main.css [hunk 2]
```diff
@@ -72,10 +73,10 @@ header nav {
 
 header li {
     display: inline;
-    padding: 1em 1em 0.5em;
 }
 
 header a {
+    padding: 1em 1em 0.5em;
```
Comment (rmartin-sc) [inline Styles/main.css:79]: The padding should be in the hyperlink as per lab 5

File: glossary.html [hunk 3]
```diff
@@ -25,21 +28,20 @@
             <nav>
                 <ul>
                     <li>
-                        <p><a href=index.html>Home</a></p>
+                        <a href=index.html>Home</a>
                     </li>
```
Comment (rmartin-sc) [inline glossary.html:32]: I didn't catch this from lab 1 (woops!)

File: glossary.html [hunk 4]
```diff
@@ -58,19 +60,18 @@ <h1>Glossary</h1>
...
+                Roads? Where We’re Going,
                     We Don’t Need Roads.<br>
                     <cite>—<span class="author">Dr Emmett Brown</span></cite>
             </blockquote><br>
-            © 2020 <a href="mailto: 19031654@saultcollege.ca?subject=Message%20from%20CSD120%20Website">Rudy Walling</a>
+            <p>© 2020 <a href="mailto:19031654@saultcollege.ca?subject=Message%20from%20CSD120%20Website">Rudy Walling</a>
             </p>
         </footer>
-        <a href="#top">Back To Top ◤</a>
+        <a href="#top" title="Back to top">◤</a>
```
Comment (rmartin-sc) [inline glossary.html:74]: A few more clean-up things from an earlier lab

File: lorem.html [hunk 5]
```diff
@@ -42,7 +42,8 @@
         </header>
         <main>
             <h1>Lorem Ipsum</h1>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero.
+            
+            <p><img src="images/lorem1.jpg" alt="Old cars near an old building" style="float:left; margin:1em">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero.
                 Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet.
```
Comment (rmartin-sc) [inline lorem.html:47]: Put your images right in among your text to see the full effect of floating

File: survey.html [hunk 6]
```diff
@@ -16,11 +16,8 @@
     <meta name="author" content="Rudy Walling">
     <meta name="description" content="My CSD120 2020 Website explaining who I am and what I enjoy">
     <link href="Styles/normalize.css" rel="stylesheet">
-<<<<<<< HEAD
     <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300&family=Shadows+Into+Light+Two&display=swap" rel="stylesheet">
-=======
     <link href="styles/main.css" rel="stylesheet">
->>>>>>> 9dc130d5f64e291d4b9a6fea6226b2e1fd4e713c
```
Comment (rmartin-sc) [inline survey.html:23]: Be sure to remove the conflict markers inserted by git!

**Review by RudyWall — APPROVED** [review]

**Pull request merged**